app/views/admin/settings/index.html.slim

```diff
@@ -74,7 +74,8 @@ section.block
           h2 = t("division.closed_books_date")
           = documentation_popover(@documentations_by_html,
             html_identifier: "#{form_identifier}_closed_books_date", options: popover_options)
-        p.help-block.note = t("quickbooks.closed_books_date.note")
+        - help_note_type = @division.closed_books_date.present? ? "note_set" : "note_not_set"
```

Comment:
is there a better way to choose between two different translation entries?

Reply:
This approach seems fine.